tests/test_sort.py

```diff
@@ -67,21 +64,15 @@ def test_partition(array, dtype, kth):
     ],
 )
 @pytest.mark.parametrize(
-    "dtype",
-    [numpy.float64, numpy.float32, numpy.int64, numpy.int32],
-    ids=["float64", "float32", "int64", "int32"],
+    "dtype", get_all_dtypes(no_none=True, no_bool=True, no_complex=True)
 )
 @pytest.mark.parametrize(
     "array",
-    [[1, 2, 3, 4], [-5, -1, 0, 3, 17, 100]],
+    [[1, 2, 3, 4], [-5, -1, 0, 3, 17, 100], [1, 0, 3, 0]],
     ids=[
         "[1, 2, 3, 4]",
-        "[-5, -1, 0, 3, 17, 100]"
-        # '[1, 0, 3, 0]',
-        # '[3, 2, 1, 6]',
-        # '[4, 2, 3, 3, 4, 1]',
-        # '[1, -3, 3, 0, 5, 2, 0, 1, 1, 0, 0, 1]',
-        # '[8, 2, 3, 0, 5, 2, 0, 1, 1, 3, 3, 1, 5, 2, 0, 1]'
```

Comment:
what is about other commented lines? is there any reason to completely remove from the test scope?